.gitignore

```diff
@@ -109,6 +109,7 @@ packages/sql-indexer/knexfile.dev.js
 packages/helpers/lib
 packages/common-scripts/lib
 packages/hd/lib
+packages/hd/*.json
```

Comment:
this causes `package.json` to be ignored, why we should do this

Reply:
Added test case for save/load keystore, the file name is {uuid}.json, maybe I should move the test file to another dir.